vnext/Playground/PlaygroundCpp/MainPage.h

```diff
@@ -0,0 +1,24 @@
+﻿#pragma once
+
+#include "MainPage.g.h"
+
+namespace winrt::PlaygroundCpp::implementation
+{
+    struct MainPage : MainPageT<MainPage>
+    {
+		const winrt::hstring JSFILENAME = L"App.windows";
```

Comment:
I often move the const to  first few line of MainPage.cpp and use it like this:

static constexpr auto c_jsFileName = L"App.windows";